tankersdk_identity/crypto/__init__.py

```diff
@@ -27,6 +28,17 @@ def generichash(buffer: bytes, size: int) -> bytes:
     return generichash_blake2b_salt_personal(buffer, digest_size=size)  # type: ignore
 
 
+def hashed_provisional_email(email: str) -> str:
```

Comment:
nitpick: I think I already commented on that before, but I'd have preferred `hash_provisional_email`.

We do perform an action here. Not that important though.

Reply:
Yeah I think it's like the `sorted(x)` function in python. They have their own style sometimes...